cache_system/src/backend/policy/lru.rs

```diff
@@ -888,12 +981,84 @@ async fn clean_up_loop<S>(
     }
 }
 
+/// Current element presented by the [`EvictionCandidate`].
+type EvictionCandidateCurrent<S> = Option<(Time, S, Box<dyn Any>)>;
```

Comment:
A silly naming thing, but I would find this code easier to read if this was 

```suggestion
type EvictionCandidate<S> = Option<(Time, S, Box<dyn Any>)>;
```

And then the iterator over candidates instead of being names
```
struct EvictionCandidate<S>
```

was named with `Iter`, perhaps like

```
struct EvictionCandidateIter<S>
```

Reply:
done